README.md

```diff
@@ -1 +1,14 @@
-# awesome-list
\ No newline at end of file
+# Awesome Projects [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+> A curated list of open source projects from students at KPI
+
+## List
+
+- [nano_daw](https://github.com/abelkevich/nano_daw)
+
+## License
+
+[![CC0](https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0)
```

Comment:
Currently, the repo page shows `GPL-3.0` because of the license file. I think this file needs to be replaced too to reflect this info.